oak_functions/wasm/src/tests.rs

```diff
@@ -214,9 +386,23 @@ fn create_test_wasm_state() -> WasmState<TestingLogger> {
     let wasm_module_path = oak_functions_test_utils::build_rust_crate_wasm("echo").unwrap();
     let wasm_module_bytes = std::fs::read(wasm_module_path).unwrap();
 
-    let wasm_handler = WasmHandler::create(&wasm_module_bytes, vec![testing_factory], logger)
-        .expect("couldn't create WasmHandler");
-    wasm_handler
-        .init_wasm_state(b"".to_vec())
-        .expect("couldn't create WasmState")
+    let wasm_handler = WasmHandler::create(&wasm_module_bytes[..], vec![testing_factory], logger)
+        .expect("Could not create WasmHandler.");
+    let wasm_state = wasm_handler
+        .init_wasm_state(b"2".to_vec())
+        .expect("Could not create WasmState.");
+
+    wasm_state
+}
+
+// Read the u32 value at the `address` from the Wasm memory.
+// Only needed in tests.
+fn read_u32(
+    ctx: &mut impl wasmi::AsContext,
+    memory: &mut wasmi::Memory,
+    address: AbiPointer,
+) -> Result<u32, OakStatus> {
+    let address = read_buffer(ctx, memory, address, 4).map_err(|_err| OakStatus::ErrInvalidArgs)?;
+    let address = LittleEndian::read_u32(&address);
```

Comment:
Does https://doc.rust-lang.org/std/primitive.i32.html#method.from_be_bytes not work?

Reply:
Does it make a difference?

Reply:
It avoids an extra external dependency for something that the std library already does. But I'm not sure if it's exactly the same functionality that you need here.

Reply:
Ah sorry this is a test anyways, so not really relevant after all.